app/scenes/channel_drawer/channel_drawer.js

```diff
@@ -42,8 +55,12 @@ export default class ChannelDrawer extends React.PureComponent {
                 channelMembers={channelMembers}
                 theme={theme}
                 onSelectChannel={this.selectChannel}
-                onViewChannel={this.props.actions.viewChannel}
-                handleCloseDM={this.props.actions.closeDMChannel}
+                onViewChannel={viewChannel}
```

Comment:
This would be a separate PR, but I think we need to consider making the `ChannelList` or its children bind these actions themselves. `ChannelDrawer` is ending up very tightly coupled to `ChannelList` which is something we were hoping redux would remove the need for